src/main/js/main_html5.js

```diff
@@ -1483,8 +1483,21 @@ require("../../../html5-common/js/utils/environment.js");
         this.controller.notify(this.controller.EVENTS.PAUSED);
       }
       forceEndOnPausedIfRequired();
+      removeControlsAttr();
     }, this);
 
+    /**
+     * IOS native player adds attribute "controls" to video tag.
+     * This function removes the attribute on IOS if it is necessary
+     * @private
+     * @method OoyalaVideoWrapper#removeControlsAttr
+     */
+    var removeControlsAttr = _.bind(function(){
+      if (OO.isIos && _video.hasAttribute('controls')) {
+        _video.removeAttribute('controls');
```

Comment:
Does iOS add the controls attribute back in when going back into fullscreen mode? Is the controls attribute used for anything in fullscreen?

Reply:
iOs automatically adds it. :-)